presto-main/src/main/java/com/facebook/presto/sql/planner/PlanPrinter.java

```diff
@@ -971,6 +977,28 @@ private String formatDomain(Domain domain)
         }
     }
 
+    private static String getFrameInfo(Window window)
```

Comment:
I'd call this `formatFrame` and make it take a `WindowFrame` as an argument. The Window is not used for anything other than getting the frame.